include/igl/opengl/glfw/imgui/SelectionPlugin.cpp

```diff
@@ -0,0 +1,199 @@
+#include "SelectionPlugin.h"
+
+#include <imgui/imgui.h>
+#include <imgui_impl_glfw.h>
+#include <imgui_impl_opengl3.h>
+#include <imgui_fonts_droid_sans.h>
+#include <GLFW/glfw3.h>
+
+namespace igl{ namespace opengl{ namespace glfw{ namespace imgui{
+
+IGL_INLINE void SelectionPlugin::init(igl::opengl::glfw::Viewer *_viewer)
+{
+  ImGuiMenu::init(_viewer);
+  std::cout<<R"(
+igl::opengl::glfw::imgui::SelectionPlugin usage:
+  [drag]  Draw a 2D selection
+  l       Turn on and toggle between lasso and polygonal lasso tool
+  M,m     Turn on and toggle between rectangular and circular marquee tool
+  V,v     Turn off interactive selection
+)";
+}
+IGL_INLINE bool SelectionPlugin::pre_draw() 
+{
+  if(!visible){ return false; }
+  ImGuiMenu::pre_draw();
+  return false;
+}
+IGL_INLINE bool SelectionPlugin::post_draw() 
+{
+  if(mode == OFF){ return false; }
+  ImGuiIO& io = ImGui::GetIO();
+  
+  ImGui::SetNextWindowPos( ImVec2(0,0) );
+  ImGui::Begin("testing", nullptr, ImGuiWindowFlags_NoBackground | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoInputs);
+  float width, height;
+  float highdpi = 1.0;
+  {
+    int fwidth, fheight;
+    glfwGetFramebufferSize(viewer->window, &fwidth, &fheight);
+    int iwidth, iheight;
+    glfwGetWindowSize(viewer->window, &iwidth, &iheight);
+    highdpi = float(iwidth)/float(fwidth);
+    // highdpi
+    width = (float)iwidth;
+    height = (float)iheight;
+  }
+  ImGui::SetWindowSize(ImVec2(width,height));
```

Comment:
This suggestion didn't compile:

```
/Users/ajx/Repos/libigl/include/igl/opengl/glfw/imgui/SelectionPlugin.cpp:48:56: error: use of undeclared identifier 'ImGuiSetCond_Always'; did you mean
      'ImGuiCond_Always'?
  ImGui::SetNextWindowSize(ImGui::GetIO().DisplaySize, ImGuiSetCond_Always);
                                                       ^~~~~~~~~~~~~~~~~~~
                                                       ImGuiCond_Always
/Users/ajx/Repos/libigl/external/libigl-imgui/../imgui/imgui.h:1267:5: note: 'ImGuiCond_Always' declared here
    ImGuiCond_Always        = 1 << 0,   // Set the variable
```

If I use `ImGuiCond_Always` then it compiles but drawing doesn't work.

Reply:
> If I use ImGuiCond_Always then it compiles but drawing doesn't work.

That's odd. Have you tried to enable the window border to verify that the window placement is correct? I can take a look when I have a moment otherwise.